src/base/rss/rss_article.cpp

```diff
@@ -112,6 +112,11 @@ void Article::markAsRead()
     }
 }
 
+void Article::markAsUnread()
```

Comment:
Why we need to undo it? It breaks the application logic, isn't it?
@mgziminsky, what do you say about it?